extensions/wikia/ExactTargetUpdates/tasks/ExactTargetRetrieveUserHelper.php

```diff
@@ -26,6 +26,35 @@ public function getUserEmail( $iUserId ) {
 		return null;
 	}
 
+	public function retrieveUserDataById( $iUserId ) {
+		$aProperties = [
+			'user_id',
+			'user_name',
+			'user_real_name',
+			'user_email',
+			'user_email_authenticated',
+			'user_registration',
+			'user_editcount',
+			'user_touched'
+		];
+
+		$oHelper = $this->getUserHelper();
+		$aApiParams = $oHelper->prepareUserRetrieveParams( $aProperties, 'user_id', [ $iUserId ] );
+
+		$oApiDataExtension = $this->getApiDataExtension();
+		$oUserResult = $oApiDataExtension->retrieveRequest( $aApiParams );
+		if ( isset( $oUserResult->Results->Properties->Property )
+			&& is_array( $oUserResult->Results->Properties->Property ) ) {
```

Comment:
```is_array``` will return ```true``` for empty array here. I will consider using ```!empty```

By mediawiki code conventions (http://www.mediawiki.org/wiki/Manual:Coding_conventions#Line_continuation),
brackets ```) { ``` should be in new line.